torch/_inductor/scheduler.py

```diff
@@ -1263,6 +1267,174 @@ def prune_redundant_deps(
             node.prune_redundant_deps(name_to_fused_node)
 
 
+class GroupedSchedulerNode(BaseSchedulerNode):
```

Comment:
This doesn't look like it supports fusions within the group, will you need to support this to get good perf?

Reply:
yes I believe in the long term we need fusion within the group. Updated PR to support this by calling `fuse_nodes()` on each group in `process_grouped_nodes()`.